contribs/gmf/src/directives/layertree.js

```diff
@@ -273,14 +276,14 @@ gmf.LayertreeController.prototype.getLayerCaseMixedGroup_ = function(node) {
   for (i = 0; i < subNodes.length; i++) {
     subNode = subNodes[i];
     // Create all sublayers include wms layers;
-    layer = this.getLayer(subNode, 1, true);
+    layer = this.getLayer(subNode, 1, true, node);
```

Comment:
I wonder if all of this is still usefull.
@ger-benjamin is it still usefull to add the Image layers to the group in one shot here, while we could add them when they come from the `getLayers` function ?